### PR TITLE
Fixed grey lines

### LINKED ISSUE #369: Align the "About OpenCode" Section Properly
Screenshot from 2020-02-12 19-27-26

Remove or edit the grey bars

### PR DESCRIPTION
**Pull request by sarthakm21**
__369__ 

Fixes #369 

#### Short description of what this resolves:

Removed the grey lines

#### Changes proposed in this pull request and/or Screenshots of changes:

-
grey

-
-

### CODE REVIEW

**Comment by Jigar3** [comment]
live-link?

**Comment by sarthakm21** [comment]
Sir I am not getting an option to share the live link for any branch other than the master branch

**Comment by sarthakm21** [comment]
Although the only changes that I made are removing those lines and the change is visible in the screenshot

**Comment by Jigar3** [comment]
Try netlify. In GitHub you can't create live link on any branch other than master

**Comment by sarthakm21** [comment]
https://amazing-minsky-b9a3ff.netlify.com/

**Comment by Jigar3** [comment]
The section looks really short, it could be nice if you could add more content in the about Section

**Comment by sarthakm21** [comment]
@Jigar3 Please look at the link again and tell me if I should add more content..

File: index.html [hunk 1]
```diff
@@ -107,17 +107,19 @@ <h2>Get started with your<br>Opensource Business!</span></h2>
...
-          </div>
 
-          <div class="col-lg-7 col-md-6">
+          <div>
             <div class="about-content">
               <h2>About Opencode</h2>
 
-              <p>OpenCode is a month of opensource where programmers, designers and innovators code, design, learn and build solutions that tackle real-world problems.</p>
+              <p>OpenCode is a month of opensource where programmers, designers and innovators code, design, learn and build solutions that tackle real-world problems.
+              It is organised anually by Geekhaven, the technical society of Indian Institute of Information Technology, Allahabad. It aims at giving
```
Comment (Jigar3) [inline index.html]: *annually

**Review by Jigar3 — CHANGES_REQUESTED** [review]
Check for spelling errors

**Comment by sarthakm21** [comment]
Done!!

**Pull request merged**